benchmark-framework/src/main/java/io/openmessaging/benchmark/RateController.java

```diff
@@ -68,7 +69,17 @@ class RateController {
             return nextRate(periodNanos, published, expected, publishBacklog, "Publish");
         }
 
-        rampUp();
+        if (targetP99EndToEndLatency != 0 && targetP99PublishLatency != 0) {
+            if (p99EndToEndLatency > targetP99EndToEndLatency) {
+                rampDown();
+                return rate * (targetP99EndToEndLatency / p99EndToEndLatency);
```

Comment:
Plz handle the `divided by zero` error.  same for the below.

Reply:
`targetP99EndToEndLatency != 0 && targetP99PublishLatency != 0` means that `targetP99EndToEndLatency > 0 && targetP99PublishLatency >0`, so `p99EndToEndLatency` must larger than zero.

> Plz handle the `divided by zero` error. same for the below.